modules/accessing-an-example-cluster-node-tuning-operator-specification.adoc

```diff
@@ -20,3 +20,11 @@ the {product-title} platform and any custom changes to the default CR will be
 overwritten by the Operator. For custom tuning, create your own tuned CRs.
 Newly created CRs will be combined with the default CR and custom tuning
 applied to {product-title} nodes based on node/pod labels and profile priorities.
+
+While in certain situations the support for pod labels can be a convenient
+way of automatically delivering required tuning, this practice is discouraged
```

Comment:
against, especially
matching, then the functionality

Reply:
Addrressed in https://github.com/openshift/openshift-docs/pull/21325